rclcpp/include/rclcpp/timer.hpp

```diff
@@ -33,7 +33,9 @@ namespace rclcpp
 namespace timer
 {
 
-using CallbackType = std::function<void()>;
+/// The timer callback signature returns a boolean.
+// If this callback returns true, then the timer should cancel.
```

Comment:
alright, can you think of a good synonym for `continue` (since it's a reserved C++ keyword)? I'm thinking `resume`?

Also, should I elevate the namespace of `timer::ReturnCode::resume` so that we can use `timer::resume`